tests/test_cookies.py

```diff
@@ -54,18 +54,23 @@ def test_response_base_case(self):
         self.resource = CookieResource()
         self.api.add_route(self.test_route, self.resource)
         self.simulate_request(self.test_route, method="GET")
-        self.assertIn(
-            ("set-cookie",
-                "foo=bar; Domain=example.com; httponly; Path=/; secure"),
-            self.srmock.headers)
+        value = "foo=bar; Domain=example.com; HttpOnly; Path=/; Secure"
+        upper = ("set-cookie", value) in self.srmock.headers
+        lower = ("set-cookie", value.lower()) in self.srmock.headers
```

Comment:
Domain and Path are capitalized in both CPython 3.4.2 and 3.4.3; it's only HttpOnly and Secure that change.